### PR TITLE
Replace tokenValue by token to unify the parameter name

### PR DESCRIPTION
**Pull request by Cruiser13**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | not yet
| Fixed tickets | #100
| License       | MIT

Basically the same as #101 but with bc compatibility and targeting 1.4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ composer.lock
 phpunit.xml
 clover.xml
 var
+/.idea
```
Comment (vincentchalamon) [inline .gitignore:10]: This should not be set here, please configure a global .gitignore file on your computer: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files

Here is a sample: https://gist.github.com/vincentchalamon/d5defad563ed49d9306a4aa57dfd4498

File: EventListener/RequestEventListener.php [hunk 2]
```diff
@@ -105,7 +105,11 @@ public function getTokenFromRequest(KernelEvent $event): void
             return;
         }
 
-        $token = $this->passwordTokenManager->findOneByToken($request->attributes->get('tokenValue'));
+        // BC
+        if (!$request->attributes->has('token')) {
```
Comment (vincentchalamon) [inline EventListener/RequestEventListener.php:109]: `token` is always defined by the route attributes. To ensure backward compatibility, for example if anybody using this bundle is refering to `tokenValue` old attribute name, you should backport it:
```suggestion
        if (!$request->attributes->has('tokenValue')) {
```

File: EventListener/RequestEventListener.php [hunk 3]
```diff
@@ -105,7 +105,11 @@ public function getTokenFromRequest(KernelEvent $event): void
...
+            $request->attributes->set('token', $request->attributes->get('tokenValue'));
```
Comment (vincentchalamon) [inline EventListener/RequestEventListener.php:110]:
```suggestion
            $request->attributes->set('tokenValue', $request->attributes->get('token'));
```

**Review by vincentchalamon — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Cruiser13) [reply .gitignore]: Removed, we handle that per repository at our agency

File: EventListener/RequestEventListener.php (hunk 2 above)
Comment (Cruiser13) [reply EventListener/RequestEventListener.php]: Done

**Comment by vincentchalamon** [comment]
You just need to fix the tests and then it'll be perfect :ok_hand:
Thanks for your contribution!

**Comment by Cruiser13** [comment]
@vincentchalamon not sure about the test setup/coverage here. Could you check?

**Comment by Cruiser13** [comment]
Closing this as renaming tokenValue to token was not intended as discussed in #100

**Pull request closed without merging**